apps/protocol-frontend/src/pages/Redirect.tsx

```diff
@@ -0,0 +1,28 @@
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import { Flex, Spinner } from '@chakra-ui/react';
+
+const TIMEOUT = 3_000;
```

Comment:
We can perhaps drop this if we think it's a bit too long.

@alexkeating what do you think?